### PR TITLE
Feature validation

### PR DESCRIPTION
**Pull request by DuongNguyen3c13**

### CODE REVIEW

File: app/Http/Controllers/UsersController.php [hunk 1]
```diff
@@ -19,7 +19,20 @@ public function index($id = null) {
             return $this->show($id);
         }
     }
-
+    public function uploadImage(Request $request) {
+        if($request->file('file')!==null) :
+            $image = $request->file('file');
+            $imageName = $image->getClientOriginalName();
+            $destinationPath = public_path('/image');
+            $width = 426;
```
Comment (kakeyang) [inline app/Http/Controllers/UsersController.php:27]: You should define these values as const. Then readability and maintainability can increase.

File: app/Http/Controllers/UsersController.php [hunk 2]
```diff
@@ -19,7 +19,20 @@ public function index($id = null) {
             return $this->show($id);
         }
     }
-
+    public function uploadImage(Request $request) {
+        if($request->file('file')!==null) :
```
Comment (kakeyang) [inline app/Http/Controllers/UsersController.php:23]: This way to write if statement is not recommended PSR

File: public/app/app.js [hunk 3]
```diff
@@ -46,14 +58,48 @@ $scope.save = function(modalstate, id) {
...
     });
 }
 
+// show modal form
+$scope.toggle = function(modalstate, id) {
+	$scope.modalstate = modalstate;
+
+	switch (modalstate) {
+		case 'add':
+		$scope.form_title = "Add New User";
```
Comment (kakeyang) [inline public/app/app.js:79]: You'd better leave indentation here too for better readability.

File: public/app/app.js [hunk 4]
```diff
@@ -46,14 +58,48 @@ $scope.save = function(modalstate, id) {
...
+		$('#photo-format-error').html("");
+		$('#photo-size-error').html("");
+		$scope.form_title = "User Detail";
+		$scope.id = id;
+		$http.get(API_URL + 'users/' + id)
+		.success(function(response) {
+			console.log(response);
+			$scope.user = response;
+		});
+		break;
```
Comment (kakeyang) [inline public/app/app.js:97]: No default case?

Comment (DuongNguyen3c13) [reply public/app/app.js:97]: No other case would happen here so it's okay.

File: public/app/app.js [hunk 5]
```diff
@@ -46,14 +45,40 @@ $scope.save = function(modalstate, id) {
     	headers: {'Content-Type': 'application/x-www-form-urlencoded'}
     }).success(function(response) {
     	console.log(response);
-    	// $location.path('').replace();
-    	// location.reload();
+    	//reload page
+    	$http.get(API_URL + "users")
+			.success(function(response) {
+				$scope.users = response;
```
Comment (quanghoan) [inline public/app/app.js]: Pls check the identation here

File: public/app/app.js [hunk 6]
```diff
@@ -64,7 +89,11 @@ $scope.confirmDelete = function(id) {
 		}).
 		success(function(data) {
 			console.log(data);
-			location.reload();
+			//reload page
+    		$http.get(API_URL + "users")
+				.success(function(response) {
```
Comment (quanghoan) [inline public/app/app.js]: Pls remove console.log() when make PRs.

**Pull request merged**